packages/ursula/config/index.js

```diff
@@ -38,15 +39,16 @@ export default {
     gussetRatio: 0.7, // Relationship between front and back gusset widths
 
     // Percentages
-    gussetWidth: { pct: 7.2, min: 4, max: 12 }, // Gusset width in relation to seat
+    gussetWidth: { pct: 7.2, min: 2, max: 12 }, // Gusset width in relation to seat
     gussetLength: { pct: 12.7, min: 10, max: 16 }, // Gusset length in relation to seat
-    fabricStretch: { pct: 15, min: 5, max: 25 },
-    rise: { pct: 46, min: 30, max: 100 },
+    fabricStretch: { pct: 15, min: 0, max: 100 }, // temporary extension for testing
```

Comment:
Maybe you meant to remove this?

Reply:
Oh, that comment was not as descriptive as I thought... I think we *do* need to extend the range, though probably not quite this far. While the pattern is (hopefully) being tested, I'd like to leave it like this.

Reply:
Do you mean tested while reviewing this PR?

Reply:
I just found the thread about this in Discord. I get it now.